R/smoothness.R

```diff
@@ -12,6 +18,13 @@
 #' plot(x)
 #' smoothness(x, method = "cor")
 #' smoothness(x, method = "diff")
+#'
+#' # A boootstrapped value can also be computed
+#' smoothness(x, iterations = 100)
+#'
+#' # When perfectly linear, the "smoothness" is 1
+#' smoothness(1:10)
```

Comment:
The examples in the current docs report smoothness < 1 with "cor" and > 1 with "diff". Can you add some explanations on how to interpret those?

Reply:
indeed, actually the "diff" method had a reversed interpretation. I fixed accordingly (but that's a breaking change)

Reply:
Thanks, this requires a bullet point in NEWS and fixing the tests

Reply:
> Thanks, this requires a bullet point in NEWS and fixing the tests

![image](https://github.com/easystats/datawizard/assets/26301769/6dddf000-1536-4fcd-afc1-f68ed9fc39eb)